api/src/lib/token.js

```diff
@@ -0,0 +1,23 @@
+const jsonWebToken = require('jsonwebtoken')
+
+module.exports = class Token {
+  constructor (deps) {
+    this.config = deps(Config)
+    this.secret = this.config.data.getIn(['api', 'secret'])
+  }
+
+  get (prefix, duration) {
+    return jsonWebToken.sign({
+      prefix: prefix,
+      exp: String(Date.now() + duration)
```

Comment:
not that we'll want that in the long term, but for now we want to set tokens with durations of Infinity